src/sentry/api/endpoints/group_events.py

```diff
@@ -97,7 +97,7 @@ def _get_events_snuba(
         if environments:
             params["environment"] = [env.name for env in environments]
 
-        full = request.GET.get("full", False)
```

Comment:
So IIUC, the issue was that we were pulling a string rather than a boolean, and any non-empty string is truthy?If so, mind adding that to the PR description?

Reply:
Correct! Updated the description